sound/soc/sof/intel/hda-dsp.c

```diff
@@ -469,26 +469,35 @@ static int hda_dsp_set_D0_state(struct snd_sof_dev *sdev,
 
 		if (hda_dsp_d0i3_streaming(sdev))
 			flags |= HDA_PM_PG_STREAMING;
+
```

Comment:
@RanderWang From the commit message, are you saying that the order was wrong on IPC3, too?

Reply:
yes, but it didn't make negative effect